organization/team.md

```diff
@@ -50,6 +50,12 @@ organizing Google Summer of Code for NumFOCUS.
 | Phil Elson         |                 |                |
 | Thomas Caswell     |                 |                |
 
+#### Pandas
+
+| Name                | Melange Profile | GitHub Profile |
+|---------------------|-----------------|----------------|
+| Jeff Reback         |                 |                |
```

Comment:
@shoyer and @jreback How many organizations you are affiliate for GSoC? Because I can't invite you to NumFOCUS. :-)